src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/SqlFileStreamTest/SqlFileStreamTest.cs

```diff
@@ -260,40 +265,47 @@ private static void DropFileStreamDb(ref string fileStreamDir, string connString
             }
         }
 
-        private static string SetupTable(SqlConnection conn)
+        private static string SetupTable(string connString)
         {
             // Generate random table name
             string tempTable = "fs_" + Guid.NewGuid().ToString().Replace('-', '_');
-
             // Create table
             string createTable = $"CREATE TABLE {tempTable} (EmployeeId INT  NOT NULL  PRIMARY KEY, Photo VARBINARY(MAX) FILESTREAM  NULL, RowGuid UNIQUEIDENTIFIER NOT NULL ROWGUIDCOL UNIQUE DEFAULT NEWID() ) ";
-            ExecuteNonQueryCommand(createTable, conn);
+            ExecuteNonQueryCommand(createTable, connString);
 
             // Insert data into created table
             for (int i = 0; i < s_insertedValues.Length; i++)
             {
                 string prepTable = $"INSERT INTO {tempTable} VALUES ({i + 1}, {s_insertedValues[i]} , default)";
-                ExecuteNonQueryCommand(prepTable, conn);
-            }
+                ExecuteNonQueryCommand(prepTable, connString);
+            }            
 
             return tempTable;
         }
 
-        private static void ExecuteNonQueryCommand(string cmdText, SqlConnection conn)
+        private static void ExecuteNonQueryCommand(string cmdText, string connString)
         {
-            using SqlCommand cmd = conn.CreateCommand();
-            cmd.CommandText = cmdText;
-            cmd.ExecuteNonQuery();
+            using (SqlConnection conn = new SqlConnection(connString))
+            {
+                conn.Open();
+                using SqlCommand cmd = conn.CreateCommand();
+                cmd.CommandText = cmdText;
+                cmd.ExecuteNonQuery();
+            }
         }
 
-        private static byte[] RetrieveData(string tempTable, SqlConnection conn, int len)
+        private static byte[] RetrieveData(string tempTable,String connString, int len)
```

Comment:
This smells like Java 😁 
c# style guidance is to use the language `string` keyword for built in types in the language rather than the CLR `String` type. Same for int etc.